ui/components/app/edit-gas-popover/edit-gas-popover.component.js

```diff
@@ -120,6 +120,12 @@ export default function EditGasPopover({
         dispatch(createSpeedUpTransaction(transaction.id, newGasSettings));
         break;
       case EDIT_GAS_MODES.MODIFY_IN_PLACE:
+        // TODO:  This shouldn't be required by prevents the following error:
```

Comment:
I think this will likely be handled by this change in the send slice update PR: https://github.com/MetaMask/metamask-extension/pull/11549/files#diff-bad76d7279418f3520a0ca93cfaeeb6fcbb1f0d4782f6ff942d422632a1c2780R917

Reply:
@danjm Is this confirmed?  Should I remove from here?

Reply:
Confirmed that between that change and https://github.com/MetaMask/metamask-extension/pull/11615, this bug goes away and the below if block is not needed